style.css

```diff
@@ -83,36 +75,33 @@ body {
   justify-content: space-between;
 }
 
-ul {
-  padding: 0;
-  list-style-type: none;
-}
+/* section 2 */
 
-.grid-box {
-  display: grid;
-  padding: 40px;
-  grid-row-gap: 88px;
+.section-2 {
+  margin: 24px;
+  border-radius: 24px;
+  background-color: #fff;
 }
```

Comment:
- [ ] The page is divided half of it contains the project's picture and the other half contains the project's description but I found that description took more space kindly fix it so It occupies space equally.
- [ ] Kindly fix it in all the section that content image and descritption
### your design
![your](https://user-images.githubusercontent.com/81727988/178093954-c4b0b554-1221-42c9-97eb-263a31acb4d1.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/178093961-4f461e53-494d-48a9-ac34-0a4a5193a79a.png)